conjure-java-core/src/main/java/com/palantir/conjure/java/util/ErrorGenerationUtils.java

```diff
@@ -76,27 +78,25 @@ public boolean contains(ErrorDefinition errorDefinition) {
         }
     }
 
-    public record ErrorDefinitionsByPackageAndNamespace(
-            Map<String, Map<ErrorNamespace, List<ErrorDefinition>>> packageToNamespacedErrorDefs) {
+    public record ErrorDefinitionsByPackageAndNamespace(List<NamespacedErrors> namespacedErrors) {
```

Comment:
I don't think record buys us much, using `List<NamespacedErrors> namespacedErrors` directly is pretty clear imo

Reply:
Yeah, it no longer makes sense to try an encapsulate that logic. Moved to using a method that just returns `List<NamespacedErrors>`.